tests-build/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "tests-build"
```

Comment:
Instead of adding this package, could the tests go in the existing tarpc/tests/compile_fail?

Reply:
Ah, assumed `trybuild` tests would be in the plugins crate, my bad. Yes, I believe they can go there.

Reply:
Done!